website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md

```diff
@@ -0,0 +1,400 @@
+        id VARCHAR,
+        first_name VARCHAR,
+        last_name VARCHAR
+    )
+
+    WITH (
+        external_location = 's3://YOUR_S3_BUCKET_NAME/dbt-quickstart/jaffle-shop-customers/',
+        format = 'csv',
+        skip_header_line_count=1
+
+    );
+
+    CREATE TABLE jaffle_shop.jaffle_shop_orders (
+
+        id VARCHAR,
+        user_id VARCHAR,
+        order_date VARCHAR,
+        status VARCHAR
+
+    )
+
+    WITH (
+        external_location = 's3://YOUR_S3_BUCKET_NAME/dbt-quickstart/jaffle-shop-orders/',
+        format = 'csv',
+        skip_header_line_count=1
+    );
+
+    CREATE TABLE jaffle_shop.stripe_payments (
+
+        id VARCHAR,
+        order_id VARCHAR,
+        paymentmethod VARCHAR,
+        status VARCHAR,
+        amount VARCHAR,
+        created VARCHAR
+    )
+
+    WITH (
+
+        external_location = 's3://YOUR_S3_BUCKET_NAME/dbt-quickstart/stripe-payments/',
+        format = 'csv',
+        skip_header_line_count=1
+
+    );
+    ```
+4. When the queries are done, you can see the following hierarchy on the query editor's left sidebar:
+
+    <Lightbox src="/img/quickstarts/dbt-cloud/starburst-data-hierarchy.png" title="Hierarchy of data in query editor" />
+
+5. Verify that the tables were created successfully. In the query editor, run the following queries:
+
+    ```sql
+    select * from jaffle_shop.jaffle_shop_customers;
+    select * from jaffle_shop.jaffle_shop_orders;
+    select * from jaffle_shop.stripe_payments;
+    ```
+
+## Connect dbt Cloud to Starburst Galaxy 
+
+1. Log in to [Starburst Galaxy](https://galaxy.starburst.io/login).
+2. If you haven’t already, set your account’s role to dbt_user. Click your email address in the upper right corner, choose **Switch role** and select **dbt_user**.
+3. Click **Clusters** on the left sidebar.
+4. Find your cluster in the **View clusters** table and click **Connection info**. Choose **dbt** from the **Select client** dropdown. Keep the **Connection information** modal open. You will use details from that model in dbt Cloud.
+5. In another browser tab, log in to [dbt Cloud](https://cloud.getdbt.com/).
+6. Create a new project in dbt Cloud. From Account settings (using the gear menu in the top right corner), click **+ New Project**.
+7. Enter a project name and click **Continue**.
+8. Choose **Trino (Starburst)** as your connection and click **Next**.
+9. Enter the **Settings** for your new project:
+    - **Host** – The **Host** value from the **Connection information** modal in your Starburst Galaxy tab.
+    - **Port** – 443 (which is the default)
+10. Enter the **Development Credentials** for your new project:
+    - **User** – The **User** value from the **Connection information** modal in your Starburst Galaxy tab. Make sure to use the entire string, including the /dbt_user part. If you don’t include it, your default role is used and that might not have the correct permissions for project development.
```

Comment:
The "dbt_user" role has not been created by the user, so we may want to change this to "accountadmin" or something more generic.

Reply:
ah, gotcha! thank you so much!